admin/tools/docker/deployment/parallel/run.sh

```diff
@@ -51,6 +51,7 @@ docker run --detach=true \
     -e "QSERV_MASTER=$MASTER" \
     $DATA_VOLUME_OPT \
     $LOG_VOLUME_OPT \
+    -v /etc/localtime:/etc/localtime:ro \
```

Comment:
Hi Fabrice

Thank you very much for the suggestion! It makes a lot of sense to me. I’ve made proper modifications to the code. Could you review them please?

It’s in :
https://github.com/lsst/qserv/pull/262/files

I have a question to you – how do I build the container out of this? Will it be built automatically? I would certainly want to pull this container into PDAC to give it a try.

Thanks,
Igor


From: Fabrice Jammes [mailto:notifications@github.com]
Sent: Thursday, November 03, 2016 3:04 AM
To: lsst/qserv
Cc: Gaponenko, Igor; Mention
Subject: Re: [lsst/qserv] Extend a configuration of the deployment startup script to ensure (#262)


In admin/tools/docker/deployment/parallel/run.sh<https://github.com/lsst/qserv/pull/262#discussion_r86311926>:

> @@ -51,6 +51,7 @@ docker run --detach=true \

>      -e "QSERV_MASTER=$MASTER" \

>      $DATA_VOLUME_OPT \

>      $LOG_VOLUME_OPT \

> +    -v /etc/localtime:/etc/localtime:ro \

@ktlim<https://github.com/ktlim>, do we have some requirements on logs timezone for LSST? Will it be UTC or locale timezone?

@iagaponenko<https://github.com/iagaponenko> , in the second case, it seems /etc/timezone should also be handled to be consistent (I think you still have UTC in this file and it may be used by some application). An example to handle both localtime and timezone inside docker seems to be available here: https://www.ivankrizsan.se/2015/10/31/time-in-docker-containers/

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/lsst/qserv/pull/262/files/7b91380ed0f58826e93ecb3a7178c7b66fe7371a#r86311926>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ANh_ZktGG3sIujmcaCQ_upsJzK5lUOrdks5q6bGQgaJpZM4Kj89T>.